src/containers/BooksForm.js

```diff
@@ -67,19 +69,22 @@ const BookForm = ({ createBook }) => {
   };
 
   return (
-    <form>
-      <label htmlFor="bookTitle">
-        Title
+    <div className="book-form-container">
+      <div className="wrapper inner-wrapper">
+        <h2 className="book-form-heading">Add New Book</h2>
+        <form>
         <input
+          className="form-control book-title"
           type="text"
           name="bookTitleInput"
           value={state.bookTitleInput}
           onChange={handleChange}
+          placeholder="Book title"
```

Comment:
- [ ] Please make this placeholder and the category one look like the ones in the design (Smaller and transparent):
![image](https://user-images.githubusercontent.com/38522578/128946659-ebe45120-582e-498d-803d-d5574d8de621.png)

Yours:
![image](https://user-images.githubusercontent.com/38522578/128946689-8ca15058-5d5b-4c24-908a-f2024ab422ce.png)